ert3/console/__init__.py

```diff
@@ -3,6 +3,7 @@
 from pathlib import Path
 import random
 import sys
+from ert3.dummy_evaluator import dummy_evaluator
```

Comment:
Let us call it `evaluator` and not `dummy` ;)

Reply:
Fixed